.gitmodules

```diff
@@ -26,3 +26,7 @@
 	path = docs/examples/quickstart/rust
 	url = https://github.com/osohq/oso-rust-quickstart.git
 	branch = main
+[submodule "docs/examples/gitclub-sqlalchemy-flask-react"]
+	path = docs/examples/gitclub-sqlalchemy-flask-react
+	url = git@github.com:osohq/gitclub-sqlalchemy-flask-react.git
```

Comment:
What's the implication of this being an SSH link vs. all the others being HTTPS?

Reply:
It was because it was private. Now it's https.